runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java

```diff
@@ -79,7 +78,7 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 
 		String option = Text.removeTags(event.getOption().toLowerCase());
 		String target = Text.removeTags(event.getTarget().toLowerCase());
-		if (isKnockedOut && nextKnockOutTick >= client.getTickCount())
+		if (nextKnockOutTick >= client.getTickCount())
 		{
 			MenuUtil.swap(client, "pickpocket", option, target);
```

Comment:
We won't be using menu util to swap in the future, this was temporary.

Reply:
Yeah. MenuManager isn't a fan of blackjacking unfortunately